examples/PyMPDATA_examples/Arabas_and_Farhat_2020/__init__.py

```diff
@@ -1 +1,10 @@
+"""
+This module contains an implementation of a paper on pricing European and
+American options using the PyMPDATA library.
+The paper is available at: [Arabas and Farhat 2020](https://doi.org/10.1016/j.cam.2019.05.023).
+
+Each notebook in this module corresponds to a figure of a table in the paper.
```

Comment:
this might be misleading, as .ipynb files are not packaged into the wheels, hence are not technically part of the module, see: https://github.com/open-atmos/PyMPDATA/blob/main/examples/MANIFEST.in

Reply:
changed in the newest commit